sharktank/tests/models/flux/flux_test.py

```diff
@@ -17,12 +17,14 @@
     export_flux_transformer_from_hugging_face,
     export_flux_transformer,
     import_flux_transformer_dataset_from_hugging_face,
+    iree_compile_flags,
```

Comment:
Keep the compile flags in this test file, or maybe add a `compile_model.py` alongside `export.py` so there is a standard way of compiling the model that the test can call into.

See also my other comment about the tools requiring less effort to use, along with issues like https://github.com/nod-ai/shark-ai/issues/691. Generally speaking, tests should be simple: https://testing.googleblog.com/2014/07/testing-on-toilet-dont-put-logic-in.html. If the test isn't simple, the system being tested should be changed first. The software we ship (sharktank / shortfin / shark-ai) should know how to compile Flux. These flags in a test show that the software still needs help with that.

Reply:
I agree that refactoring is required, but I would like to not include it in this test.